biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -0,0 +1,1172 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You will notice a number of files and folders in the data folder. We will be working with the file, `thrombin_with_ligands.csv`. As a first step, we will assign the location of that file to a variable, using the path.join function from the `os` library. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "data/thrombin_with_ligands.csv\n"
+     ]
+    }
+   ],
+   "source": [
+    "thrombin_file = os.path.join('data', 'thrombin_with_ligands.csv')\n",
+    "print(thrombin_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Importing the data with pandas\n",
+    "As mentioned above, the pandas library has a number of excellent tools for importing data. We will be using the `read_csv` function from the pandas library to import our data. You may have encountered csv files before. csv stands for comma separated values. This is a format that spreadsheet programs like Excel and Google Sheets can both read and write. If you open a csv file in a text editor, it simply contains rows of information, where the information is separated by commas. The information can be integers, real numbers or strings. Here are the first four rows of the csv file that we will be exploring using pandas.\n",
+    "\n",
+    "PDB ID,Method,Resolution,Structure,Ligand ID,Ligand name\n",
+    "\n",
+    "3SHC,X-RAY DIFFRACTION,1.9,Human Thrombin,B01,D-phenylalanyl-N-[(4-chloropyridin-2-yl)methyl]-L-prolinamide\n",
+    "\n",
+    "3P17,X-RAY DIFFRACTION,1.43,Thrombin,99P,D-phenylalanyl-N-(pyridin-3-ylmethyl)-L-prolinamide\n",
+    "\n",
+    "2ZNK,X-RAY DIFFRACTION,1.8,Thrombin,31U,D-leucyl-N-(4-carbamimidoylbenzyl)-L-prolinamide\n",
+    "\n",
+    "If you look at these data carefully, you can see how they could easily be arranged in a table, with the labels in the in the first row as the column headers. Before we can explore how pandas can do that for us, let's look at the help command for the read_csv function in pandas.\n",
+    "\n",
+    "```python\n",
+    "help(pd.read_csv)\n",
+    "```\n",
+    "\n",
+    "Notice the syntax for the command:  help is followed by parentheses, which contain the function name that we want to explore. Also note that pandas is abbreviated as pd. That may seem strange at first, but you will get used to library abbreviations in short order if you work much with Jupyter notebooks, or with python in general.\n",
+    "\n",
+    "If you enter this command in a cell, the output will be over 200 lines long. You can learn about all of the arguments that can be attached to the read_csv function and how they affect the dataframe. Just to give you a taste, here are the first few lines you would see.\n",
+    "\n",
+    "Help on function read_csv in module pandas.io.parsers:\n",
```

Comment:
Surround output of help in backticks to distinguish this output from your narrative voice.